### PR TITLE
Add Jinja2 and MarkupSafe

### PR DESCRIPTION
**Pull request by msabramo**
```javascript
_ = languagePluginLoader.then(() => {
  // pyodide is now ready to use...
  pyodide.loadPackage('Jinja2').then(() => {
    pyodide.runPython(`
import jinja2

template = jinja2.Template('Hello {{ name }}!')
print(template.render(name='Zach'))
    `);
  });
});
```

### CODE REVIEW

**Comment by msabramo** [comment]
@madhur-tandon: Does this work for you?

**Comment by msabramo** [comment]
The import of `markupsafe` needs to be lowercase. I just amended and force-pushed with that change. 🤞

**Comment by msabramo** [comment]
Screen Shot 2019-03-26 at 11 48 55 AM

File: test/test_jinja2.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+def test_svg(selenium):
```
Comment (mdboom) [inline test/test_jinja2.py]:
```suggestion
def test_jinja2(selenium):
```

Comment (msabramo) [reply test/test_jinja2.py]: oops :smile:

Comment (msabramo) [reply test/test_jinja2.py]: Fixed that and also added a line to `CHANGELOG.md`.

**Pull request merged**